help/de/Help_ch_Account-Actions.xml

```diff
@@ -924,16 +1063,17 @@
           persönlichen crontab hinzufügen:
         </para>
 
-        <para><code>0 18 * * 5 /path/to/gnucash --add-price-quotes /path/to/gnucash-filename &gt; /dev/null
-          2&gt;&amp;1</code>
+        <para><code language="cron">0 18 * * 5 /path/to/gnucash --add-price-quotes /path/to/gnucash-filename &gt;
+          /dev/null 2&gt;&amp;1</code>
         </para>
 
         <para>Für &linux;, das dbus verwendet und normalerweise vom Desktop-Manager gestartet wird um mit dconf
           oder gconf zu kommunizieren, lautet der Befehl:
```

Comment:
1. nur noch den  aktuellen Daemon nennen.
2. Kausalkette nicht  klar: Cron als Systemdienst läuft unabhängig vom Desktop, muss also einen laufenden dbus sicherstellen. Dasselbe gilt übrigens auch für remote sessions, wenn du etwa das Kurs-Update von unterwegs per ssh anstoßen willst.

Reply:
Verstehe ich dich so richtig?
Damit der Kursabruf per Cron erfolgen kann, muss `dbus` ebenfalls gestartet werden. Der Eintrag in der crontab laut dann: usw.

Reply:
Fast: muß sichergestellt sein, daß er läuft.

Normalerweise läuft ja, wenn wir am PC arbeiten, Gnome, KDE, … als Desktop-Manager und hat den bereits gestartet. Daher brauchen wir den dann im Terminal-/Konsolenfenster nicht mehr extra starten.

Aber der cron wird schon viel eher gestartet, weiß also nichts davon. Auch für ein remote login per ssh braucht keine Desktop-Umgebung zu laufen.

Reply:
ok.